process-document/src/test/kotlin/com/ritense/processdocument/service/ProcessDocumentsServiceIntTest.kt

```diff
@@ -158,4 +162,22 @@ class ProcessDocumentsServiceIntTest : BaseIntegrationTest() {
         )
     }
 
+    @Test
+    @Throws(JsonProcessingException::class)
+    fun `should retrieve active process instances name`() {
+        val request = NewDocumentAndStartProcessRequest(
+            "multi-instance-process",
+            NewDocumentRequest(
+                "house",
+                objectMapper.readTree(documentJson)
+            )
+        )
+
+        val result = runWithoutAuthorization {
+            processDocumentService.newDocumentAndStartProcess(request)
+        }
+
+        assertEquals(0, result.errors().size)
```

Comment:
The list of errors will always be `0` here. Even if there are errors. Because the `multi-instance-process` uses `asyncAfter=true`.